codebuild/bin/criterion_baseline.sh

```diff
@@ -35,21 +35,33 @@ upload_artifacts(){
 }
 
 if [ -d "third-party-src" ]; then
-  # Don't run against c.a.c.
+  echo "Not running against c.a.c."
   return 0
 fi
 
-# Fetch creds and the latest release number.
-gh_login s2n_codebuild_PRs
-get_latest_release
-AWS_S3_PATH="integv2criterion_${INTEGV2_TEST}_${LATEST_RELEASE_VER}.zip"
+# setting LOCAL_TESTING disables a check for an existing baseline.
+if [ -z "${LOCAL_TESTING:-}" ]; then
+  # Fetch creds and the latest release number.
+  gh_login s2n_codebuild_PRs
+  get_latest_release
+  # Build a specific filename for this release
+  AWS_S3_PATH="integv2criterion_${INTEGV2_TEST}_${LATEST_RELEASE_VER}.zip"
```

Comment:
Nit: I'd declare this when you declare AWS_S3_URL. Not a problem, just clearer.

Reply:
`LATEST_RELEASE_VER` isn't defined until line 46

Reply:
Yes, updated.